_tour/variances.md

```diff
@@ -32,110 +36,90 @@ case class Cat(name: String) extends Animal
 case class Dog(name: String) extends Animal
 ```
 
-Both `Cat` and `Dog` are subtypes of `Animal`. The Scala standard library has a generic immutable `sealed abstract class List[+A]` class, where the type parameter `A` is covariant. This means that a `List[Cat]` is a `List[Animal]`. A `List[Dog]` is also a `List[Animal]`. Intuitively, it makes sense that a list of cats and a list of dogs are each lists of animals, and you should be able to use either of them for in place of `List[Animal]`.
-
-In the following example, the method `printAnimalNames` will accept a list of animals as an argument and print their names each on a new line. If `List[A]` were not covariant, the last two method calls would not compile, which would severely limit the usefulness of the `printAnimalNames` method.
+We can say that `Cat` is a subtype of `Animal`, and that `Dog` is also a subtype of `Animal`. That means that the following is well-typed:
 
 ```scala mdoc
-def printAnimalNames(animals: List[Animal]): Unit =
-  animals.foreach {
-    animal => println(animal.name)
-  }
+  val myAnimal: Animal = Cat("felix")
+```
 
-val cats: List[Cat] = List(Cat("Whiskers"), Cat("Tom"))
-val dogs: List[Dog] = List(Dog("Fido"), Dog("Rex"))
+What about boxes? If I have a `Box[Cat]`, is it a subtype of `Box[Animal]`, like `Cat` is a subtype of `Animal`? At first sight, it looks like that may be plausible, but if we try to do that, the compiler will tell us we have an error:
```

Comment:
"is it a subtype" is faulty parallelism — an instance can't be a subtype, only another type can be a subtype. Maybe pedantic, but I'd suggest rethinking that sentence to avoid the problem.